src/Tizen.NUI/src/internal/GaussianBlurView.cs

```diff
@@ -55,179 +81,205 @@ protected override void Dispose(DisposeTypes type)
                 swigCPtr = new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero);
             }
 
-            base.Dispose(type);
-        }
-
-        /// <since_tizen> 3 </since_tizen>
-        public class FinishedEventArgs : EventArgs
-        {
-            private GaussianBlurView _gaussianBlurView;
-
-            /// <since_tizen> 3 </since_tizen>
-            public GaussianBlurView GaussianBlurView
+            //Release your own unmanaged resources here.
+            //You should not access any managed member here except static instance.
+            //because the execution order of Finalizes is non-deterministic.
+            if (_finishedCallback != null)
             {
-                get
-                {
-                    return _gaussianBlurView;
-                }
-                set
-                {
-                    _gaussianBlurView = value;
-                }
+                FinishedSignal().Disconnect(_finishedCallback);
```

Comment:
this disconnecting signal should be placed above the delete_GaussianBlurView(swigCPtr), because the signal must disconnected before the native object is actually deleted. please check again.

Reply:
DONE